test/functions/vass/ref-intent-bug-2big.chpl

```diff
@@ -43,8 +43,8 @@ config const ldx=2, ldy=3;
 type elType = real;
 
 // for neighbor-cache pointers
-var auxArr: [1..1] elType;
-type cacheType = auxArr._value.type; // a class type, so it can be nil
+var auxArr: [1..1, 1..1] elType;
+type cacheType = auxArr[1,..]._value.type; // a class type, so it can be nil
```

Comment:
This test (and the bigger test that it stemmed from) is arguably a bit goofy, and both @benharsh and I were tempted to just retire it, but felt that keeping it running was the path of least resistance.  I believe what's happening here is that the test is saying "I want to store an array object field representing the result of a slice directly, but I don't know what to call that type, so I'll write a little piece of code that will resemble the operation closely enough that I can use Chapel's type query machinery to get its type.

Arguably, no "real" tests should be using _value at all (or we should improve the language), which makes it hard for me to take this test very seriously.